lib/t_web/channels/feed_channel.ex

```diff
@@ -273,10 +272,13 @@ defmodule TWeb.FeedChannel do
     {:noreply, socket}
   end
 
-  # TODO refactor
-  defp render_feed_item(feed_item, screen_width) do
-    {%FeedProfile{} = profile, distance} = feed_item
-    assigns = [profile: profile, screen_width: screen_width, distance: distance]
+  def handle_info({Accounts, :feed_filter_updated, user_id}, socket) do
+    feed_filter = Feeds.get_feed_filter(user_id)
```

Comment:
this would result in a race condition with read replicas, I suggest broadcasting feed filters